src/DataFixtures/Faker/Provider/FileUploadProvider.php

```diff
@@ -58,4 +40,32 @@ public function uploadFile(string $path, string $property, string $className)
 
         return $path;
     }
+
+    /**
+     * Upload a file.
+     *
+     * @param string $path        the file path relative to the fixtures directory
+     * @param string $destination the destination path. Can be a directory or a filename.
+     *
+     * @return string
+     */
+    public function copyFile(string $path, string $destination)
```

Comment:
At first glance it seems a bit strange that the `copyFile` method uploads a file but then again i suppose that is what is needed when creating fixtures. Maybe a little comment would make it clearer.